src/mnode/src/mnodeSdb.c

```diff
@@ -549,6 +549,14 @@ static int32_t sdbInsertHash(SSdbTable *pTable, SSdbRow *pRow) {
   }
 
   pthread_mutex_lock(&pTable->mutex);
+  void **ppRow = (void **)taosHashGet(pTable->iHandle, key, keySize);
+  if (ppRow) {
+    SSdbRow row = {
```

Comment:
I prefer a test case that can prove the memory leak and validate this PR. How about create a test case to prove the problem at first?

Reply:
Please refer to line 282 in src/util/src/hash.c